### PR TITLE
feat: Minor performance improvement and logs for tracking [WPB-10738]

### PR DESCRIPTION
**Pull request by e-maad**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-10738" title="WPB-10738" target="_blank">SpikeWPB-10738</a>  [Web] Investigate possible improvements to initial decryption of messages upon loading the app [timebox: 3 days]
  </td></table>
  <br />
 

## Description
Minor performance improvement and logs for tracking 

## Screenshots/Screencast (for UI changes)

https://github.com/user-attachments/assets/9edbdac0-81f3-4304-9195-9dabac433d48

## Checklist

- [x] mentions the JIRA issue in the PR name (Ex. [WPB-423])
- [x] PR has been self reviewed by the author;
- [ ] Hard-to-understand areas of the code have been commented;
- [ ] If it is a core feature, unit tests have been added;

[WPB-423]: https://wearezeta.atlassian.net/browse/WPB-423?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/script/main/app.ts [hunk 1]
```diff
@@ -372,6 +372,8 @@ export class App {
       const selfUser = await this.repository.user.getSelf([{position: 'App.initiateSelfUser', vendor: 'webapp'}]);
 
       await initializeDataDog(this.config, selfUser.qualifiedId);
+      const eventLogger = new InitializationEventLogger(selfUser.id);
+      eventLogger.log('app-initialize');
```
Comment (przemvs) [inline src/script/main/app.ts]: Can we make ENUM for this values? 🤔

File: src/script/main/app.ts [hunk 2]
```diff
@@ -495,16 +498,19 @@ export class App {
...
+      eventLogger.log('setup-mls');
       telemetry.timeStep(AppInitTimingsStep.UPDATED_FROM_NOTIFICATIONS);
       telemetry.addStatistic(AppInitStatisticsValue.NOTIFICATIONS, totalNotifications, 100);
-
       onProgress(97.5, t('initUpdatedFromNotifications', this.config.BRAND_NAME));
 
       const clientEntities = await clientRepository.updateClientsForSelf();
-      await eventTrackerRepository.init(propertiesRepository.getUserConsentStatus().isTelemetryConsentGiven);
+
+      // not waiting for promise to unblock app initialization
```
Comment (aweiss-dev) [inline src/script/main/app.ts]: We don't really unblock it. It just appears to the user that the app is done. The user still needs to wait until every message is loaded before being able to see a call, etc. :/

Comment (e-maad) [reply src/script/main/app.ts]: Would you like to suggest a better comment?

File: src/script/main/app.ts (hunk 1 above)
Comment (e-maad) [reply src/script/main/app.ts]: I gave it a thought, but we are not using it anywhere else, is it fine to use enum?
Open for suggestions.

File: src/script/main/app.ts (hunk 2 above)
Comment (aweiss-dev) [reply src/script/main/app.ts]: We unblock the lock screen by loading this code asynchronously, to make it appear to the user that the app is done loading earlier.

**Review by V-Gira — APPROVED** [review]

**Review by aweiss-dev — APPROVED** [review]

**Review by przemvs — APPROVED** [review]

**Pull request merged**